resources/dicts/thoughts/dead/starclan/apprentice.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "id": "apprentice_dead_apprentice",
+        "thoughts": [
+            "Is wondering if r_c would have been their friend"
+        ],
+        "random_status_constraint": [
```

Comment:
'main_status_constaint' is missing, yes age will also do, but for all other this constaint is used. Maybe also use this constraint to stay more 'in line' with the style.